src/dash-playlist-loader.js

```diff
@@ -431,7 +433,8 @@ export default class DashPlaylistLoader extends EventTarget {
           internal: true,
           playlistExclusionDuration: Infinity,
           // MEDIA_ERR_NETWORK
-          code: 2
+          code: 2,
+          errorType: videojs.Errors.DashManifestSidxContainerError,
```

Comment:
I would name it as UnsupportedSidxContainer, and probably pass received container as additional metadata.